iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/SpecificVersionTagCheck.java

```diff
@@ -54,6 +54,10 @@ private static void checkFromInstruction(CheckContext ctx, FromInstruction fromI
     }
     String fullImageName = resolvedImage.value();
 
+    if ("scratch".equals(fullImageName)) {
```

Comment:
I see that we already have the same check in `PrivilegedUserCheck.isScratchImage`. Maybe you could extract it into `CheckUtils`?